README.md

```diff
@@ -1,19 +1,241 @@
-<!--
-SPDX-FileCopyrightText: 2019-present Open Networking Foundation <info@opennetworking.org>
+# ONOS Release Process Notes
+To release ONOS software builds you will need the following:
+* An account on ONOS Jenkins (jenkins.onosproject.org) with administrator privileges
+* An account on open source sonatype (oss.sonatype.org) with the `org.onosproject` staging profile
 
-SPDX-License-Identifier: Apache-2.0
--->
+ONOS releases are done by Continuous Integration (CI) jobs on Jenkins. There 
```

Comment:
I am wondering if this place is the best place for this doc? Another option is here which we have some generic information for developers or onos project in general. 
https://github.com/onosproject/onos-docs/tree/master/docs/content